When eprefix is defined, no host tools should be used.

diff --git a/pym/portage/package/ebuild/doebuild.py b/pym/portage/package/ebuild/doebuild.py
index 26e6123..c266c83 100644
--- a/pym/portage/package/ebuild/doebuild.py
+++ b/pym/portage/package/ebuild/doebuild.py
@@ -205,11 +205,6 @@ def _doebuild_path(settings, eapi=None):
 	overrides = [x for x in settings.get(
 		"__PORTAGE_TEST_PATH_OVERRIDE", "").split(":") if x]
 
-	prefixes = []
-	if eprefix:
-		prefixes.append(eprefix)
-	prefixes.append("/")
-
 	path = overrides
 
 	if "xattr" in settings.features:
@@ -231,9 +226,9 @@ def _doebuild_path(settings, eapi=None):
 		path.append(os.path.join(x, "ebuild-helpers"))
 	path.extend(prerootpath)
 
-	for prefix in prefixes:
-		for x in ("usr/local/sbin", "usr/local/bin", "usr/sbin", "usr/bin", "sbin", "bin"):
-			path.append(os.path.join(prefix, x))
+	prefix = eprefix if eprefix else "/"
+	for x in ("usr/local/sbin", "usr/local/bin", "usr/sbin", "usr/bin", "sbin", "bin"):
+		path.append(os.path.join(prefix, x))
 
 	path.extend(rootpath)
 	settings["PATH"] = ":".join(path)
